packages/truss/src/sections/tachyons/verticalAlign.ts

```diff
@@ -5,10 +5,10 @@ export const verticalAlign: CreateMethodsFn = () =>
   newMethodsForProp(
     "verticalAlign",
     {
-      vBase: "baseline",
-      vMid: "middle",
-      vTop: "top",
-      vBottom: "bottom",
+      vaBl: "baseline",
+      vam: "middle",
+      vat: "top",
+      vaBm: "bottom",
```

Comment:
@bdow wdyt of this `vaBm` vs. `vaBottom`? Usually when we have "two `vab`s", we spell out `vaBottom` vs. `vaBaseline`, but was thinking maybe this would be cute...

Reply:
I'm indifferent, but would lean towards spelling it out. Feels more clear that way.